nav_msgs/Path crude_path
---
nav_msgs/Path optimised_path